literatur.bib

```diff
@@ -6159,6 +6159,13 @@ @InProceedings{fuentetaja-et-al-ipc2018
   crossref =     "ipc2018"
 }
 
+% TODO: Add pages
+@InProceedings{funkquist-et-al-kr2024,
+  author =       "Martin Funkquist and Simon Ståhlberg and Hector Geffner",
```

Comment:
The å needs to be escaped, see other entries for Simon.

Reply:
Done